doc/whats_new/v0.22.rst

```diff
@@ -107,6 +107,11 @@ Changelog
   preserve the class balance of the original training set. :pr:`14194`
   by :user:`Johann Faouzi <johannfaouzi>`.
 
+- |Efficiency| :func:`ensemble.MissingIndicator.fit_transform` the
+  _get_missing_features_info function is now called once when calling
+  fit_transform for MissingIndicator class. :pr:`14356` by :user:
+  `Harsh Soni <harsh020>`.
```

Comment:
This doesn't render correctly with space before it. It must be right after the `:`

Reply:
@jnothman I did open a PR, kindly review it, https://github.com/scikit-learn/scikit-learn/pull/15027